### PR TITLE
Change the index page to numerically sort into a table with statuses

### PR DESCRIPTION
**Pull request by rtyler**

### CODE REVIEW

**Comment by rtyler** [comment]
selection_320

Much better IMHO

File: jep/README.adoc [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+|===
+| Status | Title
+
+| Active :smile:
+| link:1/[JEP-1: Jenkins Enhancement Proposal Format]
+
+| Draft :speech_balloon:
+| link:2/[JEP-2: Criteria for selecting "Suggested Plugins"]
+
+| Draft :information_source:
```
Comment (daniel-beck) [inline jep/README.adoc]: Draft Not Submitted?

Comment (rtyler) [reply jep/README.adoc]: This file is just auto-generated, I make no claims to the accuracy of any specific JEP 😄

Comment (daniel-beck) [reply jep/README.adoc]: Got it, commented on the source at https://github.com/jenkinsci/jep/pull/47/files#r166982163

Comment (bitwiseman) [reply jep/README.adoc]: @daniel-beck Why not just _edit_ this PR?  😝 Like so...

**Comment by rtyler** [comment]
@bitwiseman now that this works on macOS, shall we merge?

**Pull request merged**